### PR TITLE
Update BarChart and VisualsApi Tutorial 

### PR DESCRIPTION
**Pull request by ShereinDabbah**
Api version 5.0.0 changes
Specifically add docs for `getFormattingModel` new API

### CODE REVIEW

**Review by mberdugo — APPROVED** [review]

File: powerbi-docs/developer/visuals/create-bar-chart.md [hunk 1]
```diff
@@ -200,13 +197,13 @@ Replace the content of the "objects" section with the following:
 
 Save the *capabilities.json* file.
 
-Your final *capabilities* file should look like [the one in this example](https://github.com/blackleaden/PowerBI-visuals-sampleBarChart/blob/barChartTutorial/capabilities.json).
```
Comment (AviSander) [inline powerbi-docs/developer/visuals/create-bar-chart.md:203]: where did that come from ??!?!?!? (backleaden)

File: powerbi-docs/developer/visuals/create-bar-chart.md [hunk 2]
```diff
@@ -586,61 +573,107 @@ In addition to scaling, this update method also handles selections and colors. T
 
 ### Populate the properties pane
 
-The final method in the `IVisual` function is [`enumerateObjectInstances`](visual-api.md#enumerateobjectinstances-optional). This method goes through all the *"objects"* in the *capabilities.json* file (in our case, `enableAxis` and `colorSelector`) and places them inside the [**Format** pane](conditional-format.md). The object's name is available from `EnumerateVisualObjectInstancesOptions`.
+The final method in the `IVisual` function is [`getFormattingModel`](visual-api.md#getFormattingModel-optional). This method builds and return modern format pane formatting model object that contain all format pane components and properties, Then places it inside the [**Format** pane](format-pane.md).
```
Comment (AviSander) [inline powerbi-docs/developer/visuals/create-bar-chart.md:576]: 'returns'

File: powerbi-docs/developer/visuals/create-bar-chart.md (hunk 1 above)
Comment (ShereinDabbah) [reply powerbi-docs/developer/visuals/create-bar-chart.md:203]: Seems to be a worker in microsoft-akvelon

File: powerbi-docs/developer/visuals/create-bar-chart.md (hunk 2 above)
Comment (ShereinDabbah) [reply powerbi-docs/developer/visuals/create-bar-chart.md:576]: @mberdugo I think Mona will fix my English mistakes as in here

**Pull request merged**